index.js

```diff
@@ -39,6 +41,20 @@ let defaults = require('./defaults.js');
 // can be set to a number, which will effectively set the cluster
 // option to `cluster: { processes: n }`.
 //
+// `openTelemetryProvider`
+//
+// If set, Apostrophe will register it as a global OpenTelemetry tracer provider.
+// The expected value is an object, an istance of TracerProvider.
+// If the Node SDK is used in the application instead of manual configuration,
+// the provider instance is only available as a
+// private property: `sdkInstance._tracerProvider`. An issue can be opened
+// to discuss the exposure of a public getter with the OpenTelemetry developers.
+//
+// `beforeExit`
+//
+// If set, Apostrophe will invoke it (await) before exit.
+// Expected type signature is `{ beforeExit: () => Promise<void> }`.
```

Comment:
To be consistent with the tone of our documentation, which emphasizes async/await syntax over promises, just say "`beforeExit` may be an async function, will be awaited, and takes no arguments" rather than making an explicit reference to promises here (of course I know it's the same thing).